Scripts/build.swift

```diff
@@ -174,6 +174,9 @@ for rawPlatform in rawPlatforms {
     if platform.shouldTest {
         xcodeBuildArguments.append("test")
     }
+    xcodeBuildArguments.append("-test-iterations")
+    xcodeBuildArguments.append("100")
```

Comment:
This is fun. I haven't used these flags before but they enable us to run our tests _a lot_. To help limit our exposure to flakey tests I'm running every test 100 times.

Reply:
This is a new codebase, do we need to go overkill like that? 😅

Reply:
Given that we're testing asynchronous code that is potentially quite sensitive to timing issues, I think it'll be helpful!

I found some issues in my earlier implementations by running tests 100 times locally. I figured it'd be good to get CI doing the same.

Reply:
I'm just wondering if it would make more sense to have the tests themselves run something 100 times or this overkill approach where we run all of them 100 times.

At the current code volume it's a distinction without a difference so I'm cool either way.

Reply:
Agreed we don't need to defend against that future problem just yet. We’ll Get There™. Maybe 🙂